packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -288,6 +289,8 @@ export namespace CommonCommands {
 
 }
 
+const commonDecorationsStyleSheet: CSSStyleSheet | undefined = DecorationStyle.createStyleSheet();
```

Comment:
Rather than placing this in module scope, I think it would be better to have a field on the `FrontendContribution`, instead. That will allow adopters to manipulate with extensions to this class.